src/test/java/com/snack/news/controller/AdminControllerTest.java

```diff
@@ -27,21 +27,20 @@
 
 import java.lang.reflect.Method;
 import java.util.Collections;
+import java.util.List;
 import java.util.Objects;
 
+import static com.snack.news.controller.ApiUrl.Domain.NEWS;
+import static com.snack.news.controller.ApiUrl.Domain.PICKS;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyLong;
 import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.when;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'org.springframework.test.web.servlet.request.MockMvcRequestBuilders'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4962836)